web/src/settings_components.js

```diff
@@ -681,22 +681,40 @@ function enable_or_disable_save_button($subsection_elem) {
         disable_save_btn = should_disable_save_button_for_time_limit_settings(time_limit_settings);
     } else if ($subsection_elem.attr("id") === "org-other-settings") {
         disable_save_btn = should_disable_save_button_for_jitsi_server_url_setting();
+        const $button_wrapper = $subsection_elem.find(".subsection-changes-save");
+        const is_tippy = $button_wrapper.get(0)._tippy;
```

Comment:
A better variable name here might be `tippy` or `tippy_instance` since it is not a boolean and is actually a tippy instance.

Also, would be good to update this to be `$button_wrapper[0]._tippy` just so that we have it consistent with other places in the codebase.

Reply:
Looks good otherwise. Will mark it ready for "integration review" once these changes are done